Tests/UnitTests/EdgeHitProcessorTests.swift

```diff
@@ -50,9 +50,23 @@ class EdgeHitProcessorTests: XCTestCase {
     }
     let expectedHeaders = ["X-Adobe-AEP-Validation-Token": "test-int-id"]
     let experienceEvent = Event(name: "test-experience-event", type: EventType.edge, source: EventSource.requestContent, data: ["xdm": ["test": "data"]])
-    let experienceEventWithOverwritePath = Event(name: "test-experience-event", type: EventType.edge, source: EventSource.requestContent, data: ["xdm": ["test": "data", "request": ["path": "va/v1/sessionstart"]]])
+    let experienceEventWithOverwritePath = Event(name: "test-experience-event", type: EventType.edge, source: EventSource.requestContent, data: ["xdm": ["test": "data"], "request": ["path": "/va/v1/sessionstart"]])
+
+    let invalidPaths = [
+        "/va/v1/sessionstart?query=value",
+        "//va/v1/sessionstart",
+        "/va/v1//sessionstart",
+        "/va/v1/sessionstart/@test"
+    ]
+
+    let validPaths = [
```

Comment:
Add test paths which include the underscore and tilde characters after adding them to the allowed character pattern in my previous comment.